converter/convert-llama.py

```diff
@@ -96,21 +172,39 @@ def convert(modelPath, outputPath, targetFloatType):
 
     outFile.close()
 
+def convert(modelPath, outputPath, targetFloatType):
+    model_files = list(Path(modelPath).glob('*.safetensors'))
+    if len(model_files) > 0:
+        convert_safetensor(modelPath, outputPath, targetFloatType)
+        return
+    
+    model_files = list(Path(modelPath).glob('*.pth'))
+    if len(model_files) > 0:
+        convert_pth(model_files, outputPath, targetFloatType)
+        return
+    
+    print("No llama could be found in: {modelPath}")
+
 def usage():
-    print('Usage: python convert-llama.py <modelPath> <targetFloatType>')
+    print('Usage: python convert-llama.py <modelFolder> <outputFolder> <targetFloatType>')
     exit(1)
 
 if __name__ == '__main__':
-    if (len(sys.argv) < 3):
+    if (len(sys.argv) < 4):
         usage()
 
     modelPath = sys.argv[1]
-    targetFloatType = sys.argv[2]
+    outputPath = sys.argv[2]
+    targetFloatType = sys.argv[3].lower()
 
     if (not modelPath or not isFloatTypeSupported(targetFloatType)):
         usage()
 
-    modelName = modelPath.split('/')[-1]
+    if '/' in modelPath:
+        modelName = modelPath.split('/')[-1]
+    else:
+        modelName = modelPath.split('\\')[-1]
```

Comment:
I think the `os.path.basename` function would be better to extract the filename.